packages/generator-volto/generators/addon/templates/.eslintrc

```diff
@@ -27,7 +27,8 @@
     ],
     "react-hooks/rules-of-hooks": "error",
     "react-hooks/exhaustive-deps": "warn",
-    "react/react-in-jsx-scope": "off"
+    "react/react-in-jsx-scope": "off",
```

Comment:
For 18 alpha is not really required, we only need to add an entry to the upgrade guide.

Reply:
(and fix the violations)